### PR TITLE
AviaOne add RPC API GRPC

### PR DESCRIPTION
**Pull request by AviaOne**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
have these been tested with the UX that is in the docs?

can you please combine all of your PRs into one? if i accept one, every following one will have merge conflicts which will need to be resolved.

**Comment by jcstein** [comment]
why did you open **4 PRs** that are all doing the exact same thing?

**Comment by jcstein** [comment]
#902 
#901 
#899 
#898

**Comment by AviaOne** [comment]
That's a good question
I spent more than 1h30minutes to add ONLY 3 links on your Github
All commit and PR done failed return an error by email from Github
screenshot 2023-07-15
ail like this one

**Comment by jcstein** [comment]
hi! it is okay. the reason that those are failing is because you are from outside the organization. this causes the integration test that failed, to not be able to run, because you are outside the organization

so, there is nothing wrong with any of your PRs!

I will merge this one. can you confirm that you have tested running a light node against the endpoint?

https://docs.celestia.org/nodes/light-node/#start-the-light-node

**Comment by jcstein** [comment]
hi! it is okay. the reason that those are failing is because there was a bad link in the page. this causes the integration test to fail.

so, there is nothing wrong with any of your PRs!

I will merge this one. can you confirm that you have tested running a light node against the endpoint?

https://docs.celestia.org/nodes/light-node/#start-the-light-node

File: docs/nodes/mocha-testnet.mdx [hunk 1]
```diff
@@ -60,6 +60,7 @@ Below is a list of RPC endpoints you can use to connect to Mocha testnet:
 * `rpc-t.celestia.nodestake.top`
 * `celestia.cumulo.org.es`
 * `celestia-testnet-rpc.polkachu.com`
+* `https://rpc.mocha-3.celestia.aviaone.com`
```
Comment (jcstein) [inline docs/nodes/mocha-testnet.mdx:63]:
```suggestion
* `rpc.mocha-3.celestia.aviaone.com`
```

**Review by jcstein — APPROVED** [review]

**Comment by jcstein** [comment]
just tested it out. good to go!

**Pull request merged**